v63003/u_inref/gtm8787.csh

```diff
@@ -0,0 +1,57 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+# Tests that Mupip Journal -Extract=-stdout handles its termination appropriately
+#
+
+# Creating database,journal file
+setenv gtm_test_db_format "NO_CHANGE"
+source $gtm_tst/com/gtm_test_setbgaccess.csh
+source $gtm_tst/com/gtm_test_setbeforeimage.csh
+$gtm_tst/com/dbcreate.csh mumps 1>>& create.out
+if ($status) then
+	echo "DB Create Failed, Output Below"
+	cat create.out
+endif
+$MUPIP Set -Region Default -Journal=enable,on,before,file=mumps.mjl >>& jnlcreate.out
+if ($status) then
+	echo "Journal Create Failed, Output Below"
+	cat jnlcreate.out
+endif
+
+# Updating database with massive amount of information so it will take time to extract,
+# allowing the system to easily exit while extracting
+$ydb_dist/mumps -run ^%XCMD "for i=1:1:10 set ^x(i)=i"
+set t = `date +"%b %e %H:%M:%S"`
+
+echo "# Running extract and terminating in separate terminal"
+(expect -d $gtm_tst/$tst/u_inref/gtm8787.exp > expect.outx) >& xpect.dbg
+if ($status) then
+	echo "EXPECT FAILED"
+endif
+$ydb_dist/mumps -run gtm8787 >>& temp.out
```

Comment:
Now that we are using this logic a lot more, I would like a framework change that is much better. And that is to enhance getoper.csh so if it is passed "" as $2 (the after_time), it does this random string generation and waits for that to show up). That way this test (and your prior tests too) can use this framework rather than doing "mumps -run gtm8787" etc.

Reply:
Now that you have enhanced getoper.csh to take care of this, please undo gtm8779.m too (just like you undid gtm8787.m). Thanks.